docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: /en/sql-reference/functions/time-series-functions
+sidebar_position: 172
+sidebar_label: Time Series
+---
+
+# Time Series Functions
+
+Below functions are used for time series analysis.
+
+## seriesPeriodDetectFFT
+
+Finds the period of the given time series data using FFT
+Detect Period in time series data using FFT.
```

Comment:
Lines 13 and 14 essentially tell the same. Also, a dot is missing at the end of the sentence.